🐞 undefined: gqlerror after go mod vendor

### What is the issue?

If I call `go mod vendor` within a new dagger directory, then attempt to use `daggger call`, I get an `undefined: gqlerror`. 

### Dagger version

dagger v0.18.9 (docker-image://registry.dagger.io/engine:v0.18.9) darwin/arm64

### Steps to reproduce

```
% dagger init --sdk=go 
% go mod vendor
% dagger call container-echo --stringArg 'Hello, Dagger!'
...
Error logs:

✘ Container.withExec(args: ["go", "build", "-ldflags", "-s -w", "-o", "/runtime", "."]): Container! 0.9s
# dagger/daggertest                                                                                                                                                                                                                       
./dagger.gen.go:87:14: undefined: gqlerror                                                                                                                                                                                                
! process "go build -ldflags -s -w -o /runtime ." did not complete successfully: exit code: 1
...
```

If I examine the generated `dagger.gen.go` after running with `--interactive` like ` % dagger call --interactive container-echo --stringArg 'Hello, Dagger!'` looks like the import is missing

```
/src/daggertest # cat dagger.gen.go 
// Code generated by dagger. DO NOT EDIT.

package main

import (
        "context"
        "encoding/json"
        "errors"
        "fmt"
        "log/slog"
        "os"
        "sort"

        "go.opentelemetry.io/otel"
        "go.opentelemetry.io/otel/sdk/resource"
        semconv "go.opentelemetry.io/otel/semconv/v1.26.0"
        "go.opentelemetry.io/otel/trace"

        "dagger/daggertest/internal/dagger"
        "dagger/daggertest/internal/querybuilder"
        "dagger/daggertest/internal/telemetry"
)
```


### Log output

```
% dagger call container-echo --stringArg 'Hello, Dagger!'
✔ connect 0.8s
✘ load module 1m1s
! failed to serve module: input: moduleSource.asModule failed to call module "daggertest" to get functions: call constructor: process "go build -ldflags -s -w -o /runtime ." did not complete successfully: exit code: 1
│ ✔ finding module configuration 43.6s
│ ✘ initializing module 17.4s
│ ! input: moduleSource.asModule failed to call module "daggertest" to get functions: call constructor: process "go build -ldflags -s -w -o /runtime ." did not complete successfully: exit code: 1
│ │ ✘ ModuleSource.asModule: Module! 17.4s
│ │ ! failed to call module "daggertest" to get functions: call constructor: process "go build -ldflags -s -w -o /runtime ." did not complete successfully: exit code: 1
│ │ │ ✔ go SDK: load runtime 16.6s
│ │ │ ✘ asModule getModDef 16.8s
│ │ │ ! failed to call module "daggertest" to get functions: call constructor: process "go build -ldflags -s -w -o /runtime ." did not complete successfully: exit code: 1

Error logs:

✘ Container.withExec(args: ["go", "build", "-ldflags", "-s -w", "-o", "/runtime", "."]): Container! 0.9s
# dagger/daggertest                                                                                                                                                                                                                       
./dagger.gen.go:87:14: undefined: gqlerror                                                                                                                                                                                                
! process "go build -ldflags -s -w -o /runtime ." did not complete successfully: exit code: 1

Setup tracing at https://dagger.cloud/traces/setup. To hide set DAGGER_NO_NAG=1
```

--- a/cmd/codegen/generator/go/templates/src/_dagger.gen.go/module.go.tmpl
+++ b/cmd/codegen/generator/go/templates/src/_dagger.gen.go/module.go.tmpl
@@ -5,6 +5,7 @@ import (
 	"log/slog"
 	"os"
 
+	"github.com/vektah/gqlparser/v2/gqlerror"
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/trace"
 	"go.opentelemetry.io/otel/sdk/resource"

